app/src/main/java/de/westnordost/streetcomplete/quests/sidewalk/AddSidewalk.kt

```diff
@@ -75,11 +75,14 @@ class AddSidewalk : OsmElementQuestType<LeftAndRightSidewalk> {
         return roadsWithMissingSidewalks + roadsWithInvalidSidewalkTags
     }
 
+    /* Calculate when footway is too far away from the road to be considered its sidewalk.
+       It is an estimate, and we deliberately err on the side of showing the quest too often. */
     private fun getMinDistanceToWays(tags: Map<String, String>): Float =
         (
             (estimateRoadwayWidth(tags) ?: guessRoadwayWidth(tags)) +
             (estimateParkingOffRoadWidth(tags) ?: 0f) +
-            (estimateCycleTrackWidth(tags) ?: 0f)
+            (estimateCycleTrackWidth(tags) ?: 0f) +
+            1f    // absolute minimum legal sidewalk width is usually above 1.5m, but be even more pessimistic
         ) / 2f +
         4f // + generous buffer for possible grass verge
```

Comment:
and remove that, or reduce to ~2m or something.

Reply:
In the example in the picture of https://github.com/streetcomplete/StreetComplete/issues/4494#issuecomment-1277812505, the grass verge was `3.85m`. That one is `highway=secondary` though, for `highway=residential` it may probably be narrower elsewhere, although in my city it often seems to be about `3-4m` size too (as often in has trees or shrub planted there). 

E.g. this (red arrow) josm measurement othogonal to `highway=residential` shows `3.49m` verge (and `7.60m` between middle of the road to the middle of the sidewalk):

![josm2](https://user-images.githubusercontent.com/156656/195989706-ed877ad5-05e7-4b45-91b7-659ebb83a094.png)

(but I can change it whatever way you want, or you can of course change that directly if you find it more efficient)

Reply:
> What about AddCycleway::getMinDistanceToWays?

If I'm looking correctly, it seems to use same `estimateRoadwayWidth()`  functions, so would automatically use updated `const val LANE` etc.

https://github.com/streetcomplete/StreetComplete/blob/00f00f60b192bda332dae5249b5fdcd099148f29/app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt#L142-L147

Reply:
so current `+4f` seems to be OK for current `kerb+(gutter?)+grass verge` I guess?

Reply:
okay, fine